src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java

```diff
@@ -39,38 +42,49 @@
 @Extension
 public final class GlobalCachetConfiguration extends GlobalConfiguration {
 
-    private String cachetUrl;
-    private Boolean ignoreSSL;
```

Comment:
Removing those directly will cause:
- OldDataMotitor to scream and force admins to manually handle this situation.
- and not migrate the the config.

https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility

Reply:
Thanks Oliver great comment I'll fix this issue!

Reply:
Mind the `null` from old data will be silently changed to `false` (as you have replaced `boolean` by `Boolean`). Not sure if that is a problem or not.

Reply:
@olivergondza 
Since it's a `Boolean` by default it gets the value `null` therefore the `<ignoreSSL/>` field won't be shown on `com.redhat.jenkins.plugins.cachet.GlobalCachetConfiguration.xml` at all.